lib/auth/moderated_sessions_test.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package auth
+
+import (
+	"context"
+	"testing"
+
+	"github.com/gravitational/teleport/api/client/proto"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/lib/backend/memory"
+	"github.com/gravitational/teleport/lib/modules"
+	"github.com/gravitational/teleport/lib/services/local"
+	"github.com/stretchr/testify/require"
+)
+
+type testModules struct {
+	modules.Modules
+}
+
+func (m *testModules) Features() modules.Features {
+	return modules.Features{
+		ModeratedSessions: false, // Explicily turn off moderated sessions.
+	}
+}
+
+// TestUnmoderatedSessionsAllowed tests that we allow creating unmoderated sessions even if the
+// moderated sessions feature is disabled via modules.
+func TestUnmoderatedSessionsAllowed(t *testing.T) {
+	defaultModules := modules.GetModules()
+	defer modules.SetModules(defaultModules)
+	modules.SetModules(&testModules{})
+
+	bk, err := memory.New(memory.Config{})
+	require.NoError(t, err)
+
+	srv, err := local.NewSessionTrackerService(bk)
+	require.NoError(t, err)
+
+	tracker, err := srv.CreateSessionTracker(context.TODO(), &proto.CreateSessionTrackerRequest{
```

Comment:
nit: lets use `context.Background()`.

```suggestion
	tracker, err := srv.CreateSessionTracker(context.Background(), &proto.CreateSessionTrackerRequest{
```

I prefer to use `context.TODO()` only when I have the intent to go back and change it (for example, during development when initially plumbing context through an API).